notebooks/failure-type-classification/failure.pipeline

```diff
@@ -0,0 +1,149 @@
+{
+  "doc_type": "pipeline",
+  "version": "3.0",
+  "json_schema": "http://api.dataplatform.ibm.com/schemas/common-pipeline/pipeline-flow/pipeline-flow-v3-schema.json",
+  "id": "d99be5d9-d621-48de-8330-8b6687eb62d0",
+  "primary_pipeline": "5c1f75fc-9bab-4da2-bb80-2bfa9abc48de",
+  "pipelines": [
+    {
+      "id": "5c1f75fc-9bab-4da2-bb80-2bfa9abc48de",
+      "nodes": [
+        {
+          "id": "a2d8eb37-bf29-46ee-9531-14d00e50fe65",
+          "type": "execution_node",
+          "op": "execute-notebook-node",
+          "app_data": {
+            "filename": "failure_type_classifier.ipynb",
+            "runtime_image": "quay.io/aicoe/ocp-ci-analysis:latest",
+            "env_vars": [
+              "S3_ENDPOINT=http://s3.openshift-storage.svc/",
+              "S3_ACCESS_KEY=v3FnruQ78kfeULDjejUB",
+              "S3_SECRET_KEY=kJiDiHXncLJOXbaL7Zeb5Ok+gkLt9sWIa1rWAJa0",
+              "S3_BUCKET=opf-datacatalog",
+              "IN_AUTOMATION=True"
+            ],
+            "include_subdirectories": false,
+            "invalidNodeError": null,
+            "outputs": [],
+            "dependencies": [
+              "metric_template.ipynb",
```

Comment:
Can you update this line to point to the correct location of 'notebooks/data-sources/TestGrid/metrics/metric_template.ipynb' and see if this works instead of duplicating the notebook in this dir?

Reply:
This was duplicated for the pipeline dependencies. For the pipeline to run without error, it needs the notebooks that are installing the dependencies, rather than just references to other notebooks that install the dependencies. This is the current workaround, lmk if there is a better way to make this work :)

Reply:
:+1: Yes, I realized its not that simple after I made the comment. That said, I am pretty opposed to duplicating notebooks across the repo as it is an unsustainable pattern. We are looking into some possible alternatives now, but if none of them pan out we can apply this temporarily until https://github.com/elyra-ai/elyra/pull/1689 is included into the latest release.